### PR TITLE
* Work detail page to display search ui fix 

### PR DESCRIPTION
**Pull request by ishouty**
Work detail page to display search ui fix - not use ToggleContext but use GlobalContext from serverProps

* WorkHeadPrototype console error .. returning false when item is not expecting boolean with lang 
* clean up remove toggleContext which is not used in images.tsx

### CODE REVIEW

**Review by jamesgorrie — APPROVED** [review]

File: common/views/components/WorkHeaderPrototype/WorkHeaderPrototype.tsx [hunk 1]
```diff
@@ -104,7 +104,9 @@ const WorkHeaderPrototype: FunctionComponent<Props> = ({
             })}
             // We only send a lang if it's unambiguous -- better to send
             // no language than the wrong one.
-            lang={work.languages.length === 1 && work.languages[0].id}
+            lang={
+              work.languages.length === 1 ? work.languages[0].id : undefined
```
Comment (davidpmccormick) [inline common/views/components/WorkHeaderPrototype/WorkHeaderPrototype.tsx:108]: `work.languages?.[0]?.id` ?

**Review by davidpmccormick — APPROVED** [review]

File: common/views/components/WorkHeaderPrototype/WorkHeaderPrototype.tsx (hunk 1 above)
Comment (ishouty) [reply common/views/components/WorkHeaderPrototype/WorkHeaderPrototype.tsx]: cheers, ive changed this in all the areas with work.languages to work.languages[0]?.id

File: catalogue/webapp/components/Work/Work.tsx [hunk 2]
```diff
@@ -124,7 +124,7 @@ const Work: FunctionComponent<Props> = ({
       null,
     // We only send a langCode if it's unambiguous -- better to send no language
     // than the wrong one.
-    langCode: work.languages.length === 1 && work.languages[0].id,
+    langCode: work.languages.length === 1 && work.languages[0]?.id,
```
Comment (davidpmccormick) [inline catalogue/webapp/components/Work/Work.tsx:127]: why not optional chain the lot?

Comment (davidpmccormick) [reply catalogue/webapp/components/Work/Work.tsx:127]: I wondered about us enforcing the [prefer optional chaining](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-optional-chain.md) lint rule earlier. There's one for nullish coalescing too, which would probably make sense

Comment (ishouty) [reply catalogue/webapp/components/Work/Work.tsx:127]: adding linting rule for this makes total sense

**Pull request merged**